tests/release/services/missing_release_plan.go

```diff
@@ -12,7 +13,7 @@ import (
 	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
 )
 
-var _ = framework.ReleaseSuiteDescribe("[HACBS-2360] Release CR fails when missing ReleasePlan.", Label("release", "release-neg", "negMissingReleasePlan", "HACBS"), func() {
+var _ = framework.ReleaseServicesSuiteDescribe("[HACBS-2360] Release CR fails when missing ReleasePlan.", Label("release-services", "release-neg", "negMissingReleasePlan", "HACBS"), func() {
```

Comment:
Please use "ReleaseServiceSuiteDescribe" instead of "ReleaseServicesSuiteDescribe" and "release-service" instead of "release-services".